aspnetcore/mvc/models/file-uploads.md

```diff
@@ -310,7 +310,7 @@ Files uploaded using the <xref:Microsoft.AspNetCore.Http.IFormFile> technique ar
 For another example that loops over multiple files for upload and uses safe file names, see `Pages/BufferedMultipleFileUploadPhysical.cshtml.cs` in the sample app.
 
 > [!WARNING]
-> [Path.GetTempFileName](xref:System.IO.Path.GetTempFileName*) throws an <xref:System.IO.IOException> if more than 65,535 files are created without deleting previous temporary files. The limit of 65,535 files is a per-server limit. For more information on this limit on Windows OS, see the remarks in the following topics:
+> [Path.GetTempFileName](xref:System.IO.Path.GetTempFileName%2A) throws an <xref:System.IO.IOException> if more than 65,535 files are created without deleting previous temporary files. The limit of 65,535 files is a per-server limit. This limitation does not exist on operating systems other than Windows. Starting in .NET 8, the limitation does not exist on any operating system. For more information on this limit on Windows OS, see the remarks in the following articles:
```

Comment:
Tell them up front what's important. Don't make them read it if they don't need to. This note will be in effect on .NET 12 when 99% will be .NET 9 or higher. Don't penalize customers who run the current OS. Less is more. 
```suggestion
> In .NET 7 are prior, [Path.GetTempFileName](xref:System.IO.Path.GetTempFileName%2A) throws an <xref:System.IO.IOException> if more than 65,535 files are created without deleting previous temporary files. The limit of 65,535 files is a per-server limit. For more information on this limit on Windows OS, see the remarks in the following articles:
```